source/system/math.ooc

```diff
@@ -240,6 +240,15 @@ extend Float {
 		result := coefficient toString() >> "E" & power toString()
 		result
 	}
+	sineInterpolation: func -> Float {
+		result := 0.0f
+		absoluteValue := this absolute
+		if (absoluteValue > 1.0f)
+			result = 1.0f
+		else
+			result = (sin((absoluteValue - 0.5f) * pi) + 1.0f) / 2.0f
+		result
```

Comment:
Yes, was thinking about that, but thought it would be too much in one line. But since you suggest it, I will change it :-)